packages/cards-editor-middleware/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "@sap-ux/cards-editor-middleware",
+    "version": "0.1.0",
```

Comment:
Please change this to `0.2.0`. In combination with the changeset, it will then be release as `0.3.0` which is important for SAP internal usage so that it has a higher version than the internally released module with version `0.2.6`.